cypress/integration/shows.spec.ts

```diff
@@ -15,6 +15,6 @@ describe("Shows", () => {
     const showLink = cy.get('a[href*="/show/"]:first')
     showLink.click()
     cy.url().should("contain", "/show/")
-    cy.contains("Presented by")
+    cy.contains("Show")
```

Comment:
Is this relevant?

Reply:
absolutely!

Reply:
I dont know why git set it like new changes, because we have this string [here](https://github.com/artsy/force/blob/master/cypress/integration/shows.spec.ts) so it is absolutely safe